packages/validators/src/client/index.ts

```diff
@@ -43,7 +44,8 @@ export const CreateProfileFormSchema = z.object({
 
 export const EditReviewFormSchema = z.object({
   id: z.string().pipe(z.coerce.number().int().positive()),
-  restaurantId: z.string().pipe(z.coerce.number().int().positive()),
+  restaurantId: z.string(),
+  name: z.string(),
```

Comment:
```suggestion
  restaurantName: z.string(),
```
Try and restrict the value here as well (min and max lengths)

Reply:
done